lib/middleman-imageoptim/manifest_resource.rb

```diff
@@ -29,10 +29,11 @@ def ignored?
       private
 
       def manifest_content
-        if @source_file.nil?
+        path = File.join(@app.config[:build_dir], @destination_path)
+        if !File.exist?(path)
```

Comment:
Nitpick but this would be generally easier to understand using the positive form:

```
if File.exists?(path)
  do B
else
  do A
end
```

Instead of `if !x do A else do B`